src/main/java/com/kodilla/ecommercee/domain/Order.java

```diff
@@ -0,0 +1,38 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+import javax.persistence.*;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Entity(name = "order")
```

Comment:
adnotacje powinniśmy miec wszyscy ułożone wg jednego klucza. Tak masz w pozostalych encjach, tak bylo u @Jola-Witaszak , u siebie tez tak zmienię 

@Entity
@Table(name = "**ORDER_TABLE**")
@AllArgsConstructor
@NoArgsConstructor

@hubert-ros  czy getter i setter z lomboka jest w kazdej encji konieczny?

Reply:
wydaje mi się, że tu są wszystkie potrzebne adnotacje, a kolejność nie ma znaczenia. @Getter na pewno jest potrzebny, jesli chcemy korzystać z pól tej klasy.

Reply:
Kolejność ma znaczenie tylko (albo aż) estetyczne, fajnie byłoby trzymać się jednego standardu, tak samo przy opisywaniu relacji.

Reply:
Też uważam, że kolejność nie ma znaczenia, ale ok, możemy robić to według jednego klucza żeby było spójne i ładnie wyglądało 😉 Także zmieniłem. 
Wydaje mi się, że Getter będzie potrzebny, ale na razie usunąłem.